packages/opentelemetry-node/src/instrumentation/PluginLoader.ts

```diff
@@ -78,6 +78,16 @@ export class PluginLoader {
         return this;
       }
 
+      const alreadyRequiredModules = Object.keys(require.cache);
+      const requiredModulesToHook = modulesToHook.filter(name =>
+        alreadyRequiredModules.some(cached => cached.includes(name))
+      );
+      if (requiredModulesToHook.length > 0) {
+        this.logger.warn(
+          `Some modules (${requiredModulesToHook}) were already required when their respective plugin were loaded, some plugins might not work. Make sure the SDK is setup before you require in other modules.`
```

Comment:
```suggestion
          `Some modules (${requiredModulesToHook}) were already required when their respective plugin was loaded, some plugins might not work. Make sure the SDK is setup before you require in other modules.`
```

Reply:
I think we should use `requiredModulesToHook.join(', ')` to display the already loaded modules.